samples/core/getting-started/unit-testing-using-dotnet-test/README.md

```diff
@@ -2,41 +2,41 @@ Unit Testing using dotnet test Sample
 ================
 
 This sample is part of the [unit testing tutorial](https://docs.microsoft.com/dotnet/articles/core/testing/unit-testing-with-dotnet-test)
-for creating applications with unit tests included. Please see that topic for detailed steps on the code
+for creating applications with unit tests included. See that topic for detailed steps on the code
 for this sample.
 
 Key Features
 ------------
 
-This sample demonstrates creating a library and writing effective unit tests
-that validate the features in that library. 
+This sample demonstrates creating a library and writing effective unit tests that validate the features in that library. 
 
-The example provides a service that indicates whether a number is or is not
-a prime number.
+The example provides a service that indicates whether a number prime.
 
 Build and Run
 -------------
 
-To build and run the sample, change to the `src/PrimeService` directory and
-type the following three commands:
+To build and run the sample, navigate to `src/PrimeService` and run the following commands:
 
-`dotnet restore`
-`dotnet build`
-`dotnet run`
+```
+dotnet restore
+dotnet build
+dotnet run
+```
 
-`dotnet restore` installs all the dependencies for this sample into the current directory.
-`dotnet build` creates the output assembly (or assemblies).
-`dotnet run` runs the output executable. 
+`dotnet restore` installs all the dependencies.
+`dotnet build` creates the assembly (or assemblies).
+`dotnet run` runs the executable. 
 
-To run the tests, change to the `tests/PrimeService.Tests` directory and
-type the following three commands:
+To run the tests, navigate to the `tests/PrimeService.Tests` directory and type the following commands:
 
-`dotnet restore`
-`dotnet build`
-`dotnet test`
+```
+dotnet restore
+dotnet build
+dotnet test
+```
 
-`dotnet test` runs all the configure tests 
+`dotnet test` runs all the configure tests.
```

Comment:
@BillWagner what did you intend to say here initially? configured tests perhaps?